src/test/java/nl/jqno/paralleljava/TestData.java

```diff
@@ -0,0 +1,63 @@
+package nl.jqno.paralleljava;
+
+import io.vavr.collection.List;
+import io.vavr.control.Option;
+import nl.jqno.paralleljava.app.domain.PartialTodo;
+import nl.jqno.paralleljava.app.domain.Todo;
+
+import java.util.UUID;
+
+public class TestData {
```

Comment:
Should be an interface, for various reasons (one is code clutter). (Gosh, a sealed interface would be nice right about now...)

Reply:
I don't think that it necessarily _should_ be, but it sure looks a lot nicer this way.